public/signin-callback.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
```

Comment:
Don't add another html file in public folder, do this in our app using react router